protocols/luci-proto-wireguard/htdocs/luci-static/resources/protocol/wireguard.js

```diff
@@ -88,14 +95,33 @@ return network.registerProtocol('wireguard', {
 		o.validate = validateBase64;
 		o.rmempty = false;
 
+		var sections = uci.sections('network');
+		var serverName = this.getIfname();
+		var server = findSection(sections, serverName);
+
+		o = s.taboption('general', form.Value, 'public_key', _('Public Key'), _('Base64-encoded public key of this interface for sharing.'));
+		o.rmempty = false;
+
+		o.load = function(s) {
+			return getPublicKeyFromPrivate(server.private_key).then(
+				function(keypair) {
+					return keypair.pub || '';
+				}, 
+				function(error){
+					return 'Error getting PublicKey';
```

Comment:
error message is not translated

```suggestion
					return _('Error getting PublicKey');
```

Reply:
Fair point...